SPECS/nvidia-container-runtime/nvidia-container-runtime.spec

```diff
@@ -0,0 +1,63 @@
+Summary:      NVIDIA container runtime
+Name:         nvidia-container-runtime
+Version:      3.4.2
+Release:      2%{?dist}
+License:      ASL 2.0
+URL:          https://github.com/NVIDIA/nvidia-container-runtime
+Group:        Development Tools
+Vendor:       Microsoft Corporation
+Packager:     NVIDIA CORPORATION <cudatools@nvidia.com>
```

Comment:
I wasn't even aware of this tag, but we can remove it. Especially since Nvidia is probably not the packager in this case.

Reply:
removed Packager and Groups tags that were carried over from original spec files